README.md

```diff
@@ -1,7 +1,7 @@
 [![Coverage Status](https://coveralls.io/repos/github/ggravlingen/pytradfri/badge.svg?branch=master)](https://coveralls.io/github/ggravlingen/pytradfri?branch=master)
 [![PyPI version](https://badge.fury.io/py/pytradfri.svg)](https://badge.fury.io/py/pytradfri)
 
-**NB:** Latest Gateway version tested and working - 1.2.42.
```

Comment:
Didn't properly rebase my fork before starting working on this. I rebased readme manually, this is an artifact of that.